### PR TITLE
:art: Extract Cleanup Workspace in Function

### PR DESCRIPTION
**Pull request by Paulomart**
## What did you change?

Cleanup Jenkinsfile

## How can others test the changes?

> Describe how others can test your changes (you can remove this section for typo fixes etc.)

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: _templates/Jenkinsfile-node-v7 [hunk 1]
```diff
@@ -67,21 +80,10 @@ pipeline {
...
-        dir("${env.WORKSPACE}@tmp") {
-          deleteDir()
-        }
-        dir("${env.WORKSPACE}@script") {
-          deleteDir()
-        }
-        dir("${env.WORKSPACE}@script@tmp") {
-          deleteDir()
-        }
+        cleanupWorkspace();
```
Comment (cmg-dev) [inline _templates/Jenkinsfile-node-v7]: Wir hatten uns ja auf snake_case geeinigt :P 

`cleanup_workspace `

Comment (heikomat) [reply _templates/Jenkinsfile-node-v7]: @cmg-dev Die Änderung ist so übernommen worden, wie du es im skeleton bereits umgesetzt hattest: https://github.com/process-engine/skeleton/commit/977e2f32b1cf3ac8f022e5b7c332c578b09c1e2e

Comment (cmg-dev) [reply _templates/Jenkinsfile-node-v7]: @heikomat dann auch da umbauen, bitte. hat aber keine Prio.

Comment (cmg-dev) [reply _templates/Jenkinsfile-node-v7]: also issue am skeleton machen, fertig.

**Pull request merged**